cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/statements/Statement.kt

```diff
@@ -43,13 +45,14 @@ import org.neo4j.ogm.annotation.Relationship
 @NodeEntity
 abstract class Statement : Node(), DeclarationHolder {
     /**
-     * A list of local variables associated to this statement, defined by their [ ] extracted from
-     * Block because for, while, if, switch can declare locals in their condition or initializers.
+     * A list of local variables associated to this statement, defined by their
```

Comment:
doc still says only "variables" and variable declarations.

Reply:
Updated the docs